rel/i18n/emqx_license_schema.hocon

```diff
@@ -35,15 +35,19 @@ key_field.label:
 """License string"""
 
 license_root.desc:
-"""Defines the EMQX Enterprise license.
+"""~
+    Defines the EMQX Enterprise license.
 
-EMQX Enterprise is initially provided with a default trial license.
-This license, issued in April 2024, is valid for a period of 5 years.
-It supports up to 25 concurrent connections, catering to early-stage development and testing needs.
+    EMQX Enterprise is initially provided with a default trial license designed for early-stage development and testing.
+    This trial license permits up to 25 concurrent sessions across the cluster.
 
-For deploying EMQX Enterprise in a production environment, a different license is required. You can apply for a production license by visiting https://www.emqx.com/apply-licenses/emqx?version=5"""
+    Each node in the cluster has a dynamic session quota that is maintained for 30 days of continuous up-time.
+    After reaching 30 days, the session quota on that node is automatically reset to 0, and a node restart is required to reactivate the quota.
```

Comment:
Another nit. As a separate paragraph, it's not also clear this applies only to a trial license. E.g. easy to read that it's like that for any license, not just a trial one.